indico/modules/rb/client/js/modules/calendar/actions.js

```diff
@@ -139,6 +139,16 @@ export function fetchActiveBookings(limit, fetchRooms = true) {
         moment(rv.startDt, 'YYYY-MM-DD HH:mm').unix()
       ).startDt;
       params.last_reservation_id = data[lastDt][data[lastDt].length - 1].reservation.id;
+    } else if (state.linking.showNonOverlapping) {
+      params.start_dt = moment(state.linking.startDt)
+        .startOf('day')
+        .toISOString();
+    } else {
+      // Set it right before the event starts
+      params.start_dt = moment(state.linking.startDt)
+        .subtract(1, 'second')
```

Comment:
The start time is not inclusive on the server side? :o
Feels kind of strange that this is needed...

Reply:
Ah, is this because of the endless scrolling logic that loads more data? Nevermind in that case...

Reply:
Ues, that's the reason indeed.